src/Dfc.ProviderPortal.Apprenticeships/Helper/TribalHelper.cs

```diff
@@ -24,10 +24,11 @@ public TribalProvider CreateTribalProviderFromProvider(Provider provider)
         {
             var contactDetails = provider.ProviderContact.FirstOrDefault();
             var feChoice = _referenceDataServiceWrapper.GetFeChoicesByUKPRN(provider.UnitedKingdomProviderReferenceNumber).FirstOrDefault();
+            var tribalProvider = new TribalProvider();
 
             return new TribalProvider
             {
-                Id = int.Parse(provider.UnitedKingdomProviderReferenceNumber),
+                Id = tribalProvider.Id ??  int.Parse(provider.UnitedKingdomProviderReferenceNumber),
```

Comment:
You already have provider passed into the function. The id needs to come from there. This solution wouldn't work as you're creating a blank instance of TribalProvider, and it would have no data.

Reply:
Change to provider id